dev/dags/dbt/jaffle_shop/dbt_project.yml

```diff
@@ -24,3 +24,13 @@ models:
       materialized: table
       staging:
         materialized: view
+
+
+seeds:
+  jaffle_shop:
+      +meta:
+        cosmos:
+          profile_config:
+            profile_name: postgres_profile
+            profile_mapping:
+              threads: 2
```

Comment:
Shall we document this configurations

Reply:
Thanks, @pankajastro , documented in 602b64b85ce1739645d790a4f73d46e5fd3d0827